test/fixture/api/param/[id].ts

```diff
@@ -1,3 +1,4 @@
 export default eventHandler((event) => {
+  setHeader(event, "Content-Type", "text/plain; charset=utf-16");
```

Comment:
Maybe something other than plain (which is default)?

Reply:
I'm testing via the `utf-16` bit